ui/shared/components/panel/search/VariantSearchFormPanels.jsx

```diff
@@ -68,6 +68,9 @@ const ToggleHeaderFieldColumn = styled(Grid.Column)`
     >.icon {
       margin: -0.75em !important;
       transform: rotate(90deg) !important;
+      &.delete {
```

Comment:
The CSS here was not specific enough and was interacting with the x in the chit here, which is a .icon

<img width="479" alt="Screen Shot 2022-07-14 at 10 26 17 pm" src="https://user-images.githubusercontent.com/3320713/178982024-2216252f-e5c6-47cb-8416-b12bf34d8581.png">

The CSS will not affect the chit anymore, since the dropdown is no longer in the header.

I have reverted this change 👍